Gopkg.lock

```diff
@@ -553,6 +553,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "cc4d3e9fd525511c854711dd5d20dd9efe55261361ea6b77ba46cc9345e13951"
```

Comment:
this line should not change if you're not changing `Gopkg.toml` or nothing else in this file changed...

Reply:
I ran dep before commit. Is that bad? ^^

Reply:
you are right though -> it should not change.